packages/python/src/components/PropertyName.tsx

```diff
@@ -0,0 +1,29 @@
+import { MemberDeclarationContext, useContext } from "@alloy-js/core";
```

Comment:
I don't see any uses of `PropertyName`. Is this necessary?

Reply:
Nice catch, removed.